Frameworks/UIKit/UIColor.mm

```diff
@@ -715,7 +710,7 @@ - (BOOL)getHue:(float*)h saturation:(float*)s brightness:(float*)v alpha:(float*
         *v = (float)out.v;
     }
     if (a) {
-        *a = _a;
+        *a = _components.a;
     }
 
     return TRUE;
```

Comment:
Should really be YES/NO in keeping with OBJC.

---
In reply to: [71266910](https://github.com/Microsoft/WinObjC/pull/663#discussion_r71266910) [](ancestors = 71266910)